spec/data-types-and-values.html

```diff
@@ -10,17 +10,17 @@ <h1>The Record Type</h1>
       <p>The Record type is the set of all finite mappings from Strings to ECMAScript primitive values including Record and Tuple. Each record value holds an associated [[Fields]] List value which is a list of pairs of the form { [[Key]], [[Value]] } where the [[Key]] is a String and [[Value]] is any primitive value. Entries of [[Fields]] are sorted by [[Key]] in code unit order. The [[Fields]] List and its entries are never modified.</p>
 
       <emu-clause id="sec-immutable-types-record-tostring">
-        <h1>Record::toString ( _argument_ )</h1>
-        <p>The abstract operation Record::toString takes argument _argument_ (a Record) and convertes _argument_ to a String. It performs the following steps when called:</p>
+        <h1>RecordToString ( _argument_ )</h1>
+        <p>The abstract operation RecordToString takes argument _argument_ (a Record) and convertes _argument_ to a String. It performs the following steps when called:</p>
         <emu-alg>
             1. Assert: Type(_argument_) is Record.
             1. Return *"[object Record]"*.
         </emu-alg>
       </emu-clause>
 
       <emu-clause id="sec-immutable-types-record-equal">
-        <h1>Record::equal ( _x_, _y_, _elementEqual_ )</h1>
-        <p>The abstract operation Record::equal takes arguments _x_ (a Record), _y_ (a Record), and _elementEqual_ (a function object). _elementEqual_ will be invoked, with *undefined* as the receiver. It performs the following steps when called:</p>
+        <h1>RecordEqual ( _x_, _y_, _elementEqual_ )</h1>
+        <p>The abstract operation RecordEqual takes arguments _x_ (a Record), _y_ (a Record), and _elementEqual_ (a function object). _elementEqual_ will be invoked, with *undefined* as the receiver. It performs the following steps when called:</p>

```

Comment:
I missed this before, but _elementEqual_ is actually an "abstract closure", not a function object, and should be identified as such, and called with the syntax `_elementEqual_(args)` rather than with Call.